spotbugs/src/main/java/edu/umd/cs/findbugs/detect/FindDeadLocalStores.java

```diff
@@ -254,18 +262,41 @@ private void analyzeMethod(ClassContext classContext, Method method) throws Data
             BugInstance pendingBugReportAboutOverwrittenParameter = null;
             try {
                 WarningPropertySet<WarningProperty> propertySet = new WarningPropertySet<>();
+
+                InstructionHandle handle = location.getHandle();
+                int pc = handle.getPosition();
```

Comment:
I actually moved these up because the variables were declared further down and I needed them for the new code.
If that's OK I'd like to keep the existing code untouched, so the changes are limited to fixing this false positive.